src/components/user-information/user-info.tsx

```diff
@@ -30,9 +30,11 @@ export default function UserInfo(): ReactElement {
         },
     });
 
-    const [inviteCode, setInviteCode] = useState<InviteCode[]>([
+    const [orgInvitations, setInviteCode] = useState<InviteCode[]>([
         { id: "", invite_code: "", organization_id: "", created_at: "" },
     ]);
+    const inviteCode = orgInvitations[0]?.invite_code;
```

Comment:
Although capturing this in a variable makes it easier to reference / read in the markup, **it's incorrect because it's no longer reactive.**